feature/policy_forwarding/encapsulation/otg_tests/static_encap_gue_ipv4/README.md

```diff
@@ -0,0 +1,334 @@
+# PF-1.3: Policy-based Static GUE Encapsulation to IPv4 tunnel
+
+## Summary
+
+This test verifies Policy Forwarding (PF) action to encapsulate packets in an
+IPv4 GUE tunnel. The encapsulation is based on a statically configured GUE
+tunnel and static route.
+
+## Testbed Type
+
+*  [`featureprofiles/topologies/ate2_dut5_dum5.testbed`](https://github.com/openconfig/featureprofiles/blob/main/topologies/ate2_dut5_dum5.testbed)
+
+## Procedure
+
+### Test environment setup
+
+*   DUT and DUM have 5 ports each.
+
+    ```
+                              -------                  -------
+                             |       | ==== LAG1 ==== |       |
+          [ ATE:Port1 ] ---- |  DUT  |                |  DUM  | ---- [ ATE:Port2 ]
+                             |       | ==== LAG2 ==== |       |
+                              -------                  -------
+    ```
+
+*   Routes are advertised from ATE:Port2.
+*   Traffic is generated from ATE:Port1.
+*   ATE:Port2 is used as the destination port for GUE encapsulated traffic.
```

Comment:
instead,  consider ATE port 2 is just the next hop for the DUT.  The encapsulated packet can be captured at ATE port 2 and decoded to ensure it has the expected format.

Make decap testing a separate test.

Reply:
Ack, updated the test verifications. Decap will being done in a separate test.